commons-search/src/main/java/org/exoplatform/commons/search/domain/Document.java

```diff
@@ -35,6 +35,7 @@
   private String url;
   private Date lastUpdatedDate;
   private Set<String> permissions;
+  private List<String> tags;
```

Comment:
Document is a generic object used for all type of data indexed in ES, so I am not sure we should have a dedicated field for `tags`. We should probably put it in the `fields` field instead. WDYT ?

Reply:
As discussed with @ahmadmido we will keep tags as a dedicated field for the following reasons:
* `fields` only accept String values and it would require a lot of change to allow to accept any object type, which is out of the scope of this task
* in the future, most of the platform resources will have tags